x-pack/plugins/actions/docs/openapi/components/schemas/config_properties_genai.yaml

```diff
@@ -0,0 +1,12 @@
+title: Connector request properties for a generative AI connector
+description: Defines properties for connectors when type is `.gen-ai`.
+type: object
+properties:
+    apiProvider:
+        type: string
+        description: The OpenAI API provider.
+    apiUrl:
+        type: string
+        description: The OpenAI API endpoint.
+additionalProperties: true
+# TO-DO: Add the properties for this connector.
```

Comment:
looks like properties are already set, can this line be removed?

Reply:
Done, thanks!